app/views/curation_concerns/base/_relationships_collection_row.html.erb

```diff
@@ -0,0 +1,16 @@
+<tr>
+<th><%= t(".#{model_name}", type: model_name) %></th>
+<td>
+  <% if collection.blank? %>
+    <p><%= t('.empty', type: model_name) %></p>
+  <% else %>
+    <ul class="tabular">
+      <% collection.each do |col| %>
```

Comment:
If this could be something other than a collection, should it have a more descriptive name?